python-backend/app/api/permits/namespace/permits.py

```diff
@@ -1,24 +1,26 @@
 from flask_restplus import Namespace
 
-from ..permit.resources.permit import PermitResource
+from ..permit.resources.permit import PermitResource, PermitListResource
 from ..permit.resources.permit_status_code import PermitStatusCodeResource
-from ..permit_amendment.resources.permit_amendment import PermitAmendmentResource
-from ..permit_amendment.resources.permit_amendment_document import PermitAmendmentDocumentResource
+from ..permit_amendment.resources.permit_amendment import PermitAmendmentResource, PermitAmendmentListResource
+from ..permit_amendment.resources.permit_amendment_document import PermitAmendmentDocumentListResource, PermitAmendmentDocumentResource
 
 api = Namespace('permits', description='Permit related operations')
 
-api.add_resource(PermitResource, '', '/<string:permit_guid>')
+api.add_resource(PermitResource, '/<string:permit_guid>')
+api.add_resource(PermitListResource, '')
 api.add_resource(PermitStatusCodeResource, '/status-codes')
-api.add_resource(PermitAmendmentResource, '/<string:permit_guid>/amendments',
-                 '/amendments/<string:permit_amendment_guid>',
+
+api.add_resource(PermitAmendmentListResource, '/<string:permit_guid>/amendments')
+api.add_resource(PermitAmendmentResource,
                  '/<string:permit_guid>/amendments/<string:permit_amendment_guid>')
 
 api.add_resource(
-    PermitAmendmentDocumentResource,
+    PermitAmendmentDocumentListResource,
     '/amendments/documents',
```

Comment:
is this how we want to have paths available to upload files for objects no yet created?

Reply:
I couldn't think of a better path. Any suggestions? I think what we did for Incidents is a better approach overall, but that's a change in front-end behavior I'm not sure we necessarily want.

Reply:
I just moved it under mines (not pushed yet) and yes it ends up being /mines/permits/amendments/documents